src/tools/run-make-support/src/rustdoc.rs

```diff
@@ -62,6 +62,12 @@ impl Rustdoc {
         self
     }
 
+    /// Set the path where the command will be run.
+    pub fn current_dir<P: AsRef<Path>>(&mut self, path: P) -> &mut Self {
```

Comment:
Should I add this method in `Rustc` too?

Reply:
That makes sense, or even in the `impl_helper_methods` macro to generate that for all command wrappers.